data/json/recipes/recipe_electronics.json

```diff
@@ -2109,6 +2109,30 @@
   ]
 },{
   "type" : "recipe",
+  "result": "medical_laptop",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_TOOLS",
+  "skill_used": "electronics",
+  "skills_required": [ [ "computer", 4 ], [ "firstaid", 3 ] ],
+  "difficulty": 5,
+  "time": 30000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [ [ "recipe_lab_elec", 5 ], [ "recipe_medicalmut", 6 ], [ "recipe_labchem", 7] ],
+  "qualities" : [ {"id" : "SCREW", "level" : 1} ],
+  "tools": [
+    [ [ "software_medical", -1 ] ],
+    [ [ "soldering_iron", 14 ], [ "toolset", 14 ] ]
+  ],
+  "components": [
+    [ [ "laptop", 1 ] ],
+    [ [ "chemistry_set_basic", 1 ], [ "chemistry_set", 1 ] ],
```

Comment:
I think about that and I decided that's just incorrect do not allow to craft a laptop with regular chem set - yes, you will throw away a hotplate in this process but it's your (user's, I mean) choice (and it can be useful in case of total lack of basic chem set).